<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyTestClass</class>
 <widget class="QMainWindow" name="KeyTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeyTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="lblKeyTriggered">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>72</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ლ(╹◡╹ლ)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTableWidget" name="tableAccs">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Accelerator</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Key Name</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="editAccKeys">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editKeyName">
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnAdd">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnDel">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="keytest.qrc"/>
 </resources>
 <connections/>
</ui>
